package.nls.json

```diff
@@ -13,6 +13,7 @@
     "AWS.configuration.description.logLevel": "The AWS Toolkit's log level (changes reflected on restart)",
     "AWS.configuration.description.onDefaultRegionMissing": "Action to take when a Profile's default region is hidden in the Explorer. Possible values:\n* `add` - shows region in the explorer\n* `ignore` - does nothing with the region\n* `prompt` - (default) asks the user what they would like to do.",
     "AWS.configuration.description.recentCredentials": "Credentials most recently used to connect to AWS",
+    "AWS.configuration.description.samcli.debug.attach.retry.maximum": "If the Toolkit is unable to attach a debugger, this is the number of times to retry before giving up.",
```

Comment:
Maybe specify how often it retries? It's a little arbitrary right now (if anything, people would think, "Whoa, that's an awfully large number, we can turn that down to 3" or something)

Reply:
Retry attempts should be an internal concept. This gives users a dial for diagnosing attach issues, should it become necessary. 

There's also a risk of the description drifting out of sync with the actual retry delay value. Maybe we simply surface the retry delay as a configurable value (at a later date), but it shouldn't be necessary to adjust this value.